cartridges/int_klaviyo_core/cartridge/scripts/klaviyo/eventData/addedToCart.js

```diff
@@ -0,0 +1,127 @@
+                    productUPC                : basketProduct.UPC,
+                    viewedProductAvailability : basketProduct.availabilityModel.availability,
+                    categories                : categories, // was createCategories(basketProduct) in orig, check that my output from categories above matches expected output
+                    primaryCategory           : primaryCategory
+                };
+
+                if(!basketProduct.master && 'masterProduct' in basketProduct) {
+                    currentLineItem.masterProductID = basketProduct.masterProduct.ID;
+                }
+
+                var priceData = klaviyoUtils.priceCheck(lineItem, basketProduct);
+                currentLineItem.price = priceData.purchasePrice;
+                currentLineItem.priceValue = priceData.purchasePriceValue;
+                currentLineItem.originalPrice = priceData.originalPrice;
+                currentLineItem.originalPriceValue = priceData.originalPriceValue;
+
+                selectedOptions = lineItem && lineItem.optionProductLineItems ? klaviyoUtils.captureProductOptions(lineItem.optionProductLineItems) : null;
+                if (selectedOptions && selectedOptions.length) {
+                    currentLineItem.productOptions = selectedOptions;
+                }
+
+                if (lineItem.bundledProductLineItem || lineItem.bundledProductLineItems.length) {
+                    var prodBundle = klaviyoUtils.captureProductBundles(lineItem.bundledProductLineItems);
+                    currentLineItem['Is Product Bundle'] = prodBundle.isProdBundle;
+                    currentLineItem['Bundled Product IDs'] = prodBundle.prodBundleIDs;
+                }
+
+                if (lineItem.bonusProductLineItem) {
+                    currentLineItem['Is Bonus Product'] = true;
+                }
+
+                data.lineItems.push(currentLineItem);
+                data.items.push(basketProduct.name);
+                data.categories.push.apply(
+                    data.categories,
+                    data.lineItems[itemIndex].categories
+                );
+                data.primaryCategories.push(
+                    data.lineItems[itemIndex].primaryCategory
+                );
+                data.categories = klaviyoUtils.dedupeArray(data.categories);
+                data.primaryCategories = klaviyoUtils.dedupeArray(data.primaryCategories);
+            }
+        }
+
+        // Item added to the cart in this event
+        // Bonus products can occasionally appear as final item in the cart. We exclude these from the line items to accurately track the item added to cart in this event.
+        for (let i = data.lineItems.length - 1; i >= 0; i--) {
+            if (!data.lineItems[i]['Is Bonus Product']) {
+                data.productAddedToCart = data.lineItems[i];
+                break;
+            }
+        }
+
+    } catch(e) {
+        var errorTest = e;
```

Comment:
Is this var necessary or is it left behind from testing

Reply:
Thanks for catching that. Slipped under the radar. Will remove.

Reply:
good catch, we can safely remove this.